.github/workflows/ci.yml

```diff
@@ -54,6 +55,20 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --workspace --all-features --tests -- -D clippy::all
 
+      - name: Install nightly rust (docs)
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+          components: rust-docs
+          override: true
+
+      - name: Build docs
+        uses: actions-rs/cargo@v1
+        with:
+          command: doc
+          args: --workspace --all-features --no-deps
```

Comment:
Ported over your suggestion to #803 to run the cargo action so that it reports failures, which is generally a good idea. Other than that, I think this should now be equivalent.

Reply:
yep, i just wanted to try this out as i failed to do rust docs in CI in a reasonable way before